changes/2025-06-30_branch_keys_version_2/protect_branch_key_without_kms_ec.md

```diff
@@ -0,0 +1,382 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# How can the Branch Key's Context be protected without using KMS EC?
```

Comment:
What option are we going with? What are the requirements of a solution? The structure of this doc is kind of confusing so it's hard to reason about which option or combination of options we should pick.